dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/ContentletCacheImpl.java

```diff
@@ -49,122 +49,122 @@ public TranslatedQuery getTranslatedQuery(String key) {
+			}
+		}
 	}
-	
+
 	@Override
 	public String getMetadata(String key) {
-	    key = metadataGroup + key;
-	    String metadata=null;
-	    try {
-            metadata=(String)cache.get(key, metadataGroup);
-        } catch (DotCacheException e) {
-            Logger.debug(this, "Cache Entry not found", e);
-        }
-	    return metadata;
+		key = metadataGroup + key;
+		String metadata=null;
+		try {
+			metadata=(String)cache.get(key, metadataGroup);
+		} catch (DotCacheException e) {
+			Logger.debug(this, "Cache Entry not found", e);
+		}
+		return metadata;
 	}
 
 	@Override
 	public com.dotmarketing.portlets.contentlet.model.Contentlet add(String key, com.dotmarketing.portlets.contentlet.model.Contentlet content) {
 
 		addMetadata(key, content);
-	    
-	    key = primaryGroup + key;
 
-        // Add the key to the cache
-        cache.put(key, content,primaryGroup);
+		key = primaryGroup + key;
+
+		// Add the key to the cache
+		cache.put(key, content,primaryGroup);
 
 
 		return content;
-		
+
 	}
-	
+
 	@Override
 	public com.dotmarketing.portlets.contentlet.model.Contentlet get(String key) {
 		key = primaryGroup + key;
-    	com.dotmarketing.portlets.contentlet.model.Contentlet content = null;
-    	try{
-    		content = (com.dotmarketing.portlets.contentlet.model.Contentlet)cache.get(key,primaryGroup);
-    	}catch (DotCacheException e) {
+		com.dotmarketing.portlets.contentlet.model.Contentlet content = null;
+		try{
+			content = (com.dotmarketing.portlets.contentlet.model.Contentlet)cache.get(key,primaryGroup);
+		}catch (DotCacheException e) {
 			Logger.debug(this, "Cache Entry not found", e);
 		}
-        return content;	
+		return content;
 	}
 
-    /* (non-Javadoc)
-	 * @see com.dotmarketing.business.PermissionCache#clearCache()
-	 */
+	/* (non-Javadoc)
+     * @see com.dotmarketing.business.PermissionCache#clearCache()
+     */
 	public void clearCache() {
-        // clear the cache
-    	for(String group : groupNames){
-    		cache.flushGroup(group);
-    	}
-    }
-
-    /* (non-Javadoc)
-	 * @see com.dotmarketing.business.PermissionCache#remove(java.lang.String)
-	 */
+		// clear the cache
+		for(String group : groupNames){
+			cache.flushGroup(group);
+		}
+	}
+
+	/* (non-Javadoc)
+     * @see com.dotmarketing.business.PermissionCache#remove(java.lang.String)
+     */
 	public void remove(String key){
-    	
-    	String myKey = primaryGroup + key;
-    	String metadataKey = metadataGroup + key;
-    	com.dotmarketing.portlets.contentlet.model.Contentlet content = null;
-    	try {
-    		content = (com.dotmarketing.portlets.contentlet.model.Contentlet)cache.get(key,primaryGroup);
-    		try {
+
+		String myKey = primaryGroup + key;
+		String metadataKey = metadataGroup + key;
+		com.dotmarketing.portlets.contentlet.model.Contentlet content = null;
+		try {
+			content = (com.dotmarketing.portlets.contentlet.model.Contentlet)cache.get(key,primaryGroup);
+			try {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested try block into a separate method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1141)